### DISCUSSION TITLE
ScriptReference/Object.InstantiateAsync

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#622] [Notes]
# ScriptReference/Object.InstantiateAsync

https://docs.unity3d.com/ScriptReference/Object.InstantiateAsync.html

**Comment by zbigniewcebula** [comment]
Doesn't exist in 2022.3.4f1

**Reply by vertxxyz** [reply]
It was introduced in 2023.2.0a1 and 2022.3.20f1.
